web-app/src/pages/for-employers/for-employers.tsx

```diff
@@ -236,151 +392,654 @@ function ForEmployers() {
+												Mon - Fri
+											</LanguageTitle>
+											<LanguageDescription>
+												8 AM - 6 PM
+											</LanguageDescription>
+										</LanguageContentWrapper>
+									</JobDetailContent>
+								</Col>
+							</JobDetailsWrapper>
+							<JobDetailsWrapper>
+								<Col
+									xs={12}
+									md={12}
+									className="details-container"
+								>
+									<JobDetailsTitle>LOCATION</JobDetailsTitle>
+									<JobDetailContent>
+										<div>
+											<LanguageImage
+												src={cityImage}
+												alt="dollar-img"
+											/>
+											<LanguageContentWrapper border>
+												<LanguageTitle>
+													City
+												</LanguageTitle>
+												<LanguageDescription>
+													Tijuana
+												</LanguageDescription>
+											</LanguageContentWrapper>
+										</div>
+										<div>
+											<LanguageImage
+												src={industrialParkImage}
+												alt="dollar-img"
+											/>
+											<LanguageContentWrapper border>
+												<LanguageTitle>
+													Industrial Park
+												</LanguageTitle>
+												<LanguageDescription>
+													EL Logo
+												</LanguageDescription>
+											</LanguageContentWrapper>
+										</div>
+										<div>
+											<LanguageImage
+												src={addressImage}
+												alt="dollar-img"
+											/>
+											<LanguageContentWrapper
+												border={false}
+											>
+												<LanguageTitle>
+													Address
+												</LanguageTitle>
+												<LanguageDescription>
+													Calle Lagua Maynar 5520,
+													Section C
+												</LanguageDescription>
+											</LanguageContentWrapper>
+										</div>
+									</JobDetailContent>
+								</Col>
+							</JobDetailsWrapper>
+						</div>
+						<div className="description">
+							<div className="df">
+								<img src={descriptionImage} alt=""></img>
+								<span>Description</span>
+							</div>
+							<div className="tags">
+								{jobDescriptions.map((v) => {
+									return (
+										<DescriptionTag key={v}>
+											{v}
+										</DescriptionTag>
+									);
+								})}
+							</div>
+						</div>
+						<div className="description">
+							<div className="df">
+								<img src={skillsImages} alt=""></img>
+								<span>Skilld Required</span>
+							</div>
+							<div className="tags">
+								{jobSkills.map((v) => {
+									return (
+										<DescriptionTag key={v}>
+											{v}
+										</DescriptionTag>
+									);
+								})}
+							</div>
+						</div>
+						<div className="description">
+							<div className="df">
+								<img src={certificateImage} alt=""></img>
+								<span>Certificates & Licences</span>
+							</div>
+							<div className="certificates">
+								<ul>
+									{certifications.map((v) => {
+										return <li key={v}>{v}</li>;
+									})}
+								</ul>
+							</div>
+						</div>
+					</JobPostCard>
```

Comment:
Several comments here:
1. Create a separate component for JobPostCard and name it JobPost. Create the necessary props and pass in the sample data into the component.
2. There's a lot of use of divs and classNames but we should be using styled-components instead of classes and divs. Also, "df" is not a good classname but that will get fixed with good names for the styled components